workers/tasks/export/html.tsx

```diff
@@ -194,15 +198,19 @@ const renderFrontMatterForHtml = ({
 	doi,
 	title,
 	communityTitle,
+	citeAs,
 	accentColor,
 	attributions,
 	licenseSlug,
+	publisher,
 }) => {
 	const affiliations = [
 		...new Set(attributions.map((attr) => attr.affiliation).filter((x) => x)),
 	];
 	const communityAndCollectionString =
-		communityTitle + (primaryCollectionTitle ? bullet + primaryCollectionTitle : '');
+		(citeAs || communityTitle) +
+		(primaryCollectionTitle ? bullet + primaryCollectionTitle : '') +
+		(publisher ? bullet + publisher : '');
```

Comment:
Could we hoist the `citeAs` prop out of the `renderFrontMatter...` functions and compute the title (using the same `citeAs || pubData.community.title` logic) in `getPubMetadata`? That way we only have to do it in one spot, instead of both frontmatter render functions.